assets/views.py

```diff
@@ -189,3 +190,39 @@ def perform_destroy(self, instance):
         if instance.deleted_at is None:
             instance.deleted_at = now()
             instance.save()
+
+
+assets_stats_schema = ManualSchema(
+    fields=[],
+    description="Returns Assets stats: total number of assets, total number of assets completed,"
+                "total number of assets with personal data, and assets per department (total, "
+                "completed, with personal data)"
+)
```

Comment:
Can we have the schema be described in a machine-readable way so that the response type is represented properly in the swagger.json file? Like all the other endpoints. I think that can probably be done by creating a serializer for the return value.